pkg/convert/jfr/parser.go

```diff
@@ -14,155 +14,224 @@ import (
 	"github.com/pyroscope-io/pyroscope/pkg/storage/tree"
 )
 
-func ParseJFR(ctx context.Context, r io.Reader, s storage.Putter, pi *storage.PutInput) (err error) {
+type Tags struct {
```

Comment:
I know this is annoying, and we're not doing the best job of following this convention at the moment, but we're trying to switch from calling these `tags` to calling these `labels` as this is the name most other profilers adopted. Could you rename everything called `tags` to `labels`?

Reply:
renamed